include/event_scripts.h

```diff
@@ -1186,4 +1186,6 @@ extern const u8 Text_1BCA95[];
 extern const u8 Text_1BCACB[];
 extern const u8 Text_1BCAF2[];
```

Comment:
text DECLs shouldn't be here

Reply:
These point to text objects in data/event_scripts.s, not data/strings.s.

Reply:
Yes, I'm wrong. But stuff above (gText_WouldYouLikeToSaveTheGame, etc) shouldn't be here